### PR TITLE
fix(serve): rebuilds after new typings are installed

### PR DESCRIPTION
**Pull request by jkuri**

### CODE REVIEW

**Comment by hansl** [comment]
I assume this PR is going on top of PR#294?

**Comment by jkuri** [comment]
Yes. Should I include it in #294?

File: lib/broccoli/angular2-app.js [hunk 1]
```diff
@@ -63,15 +63,8 @@ Angular2App.prototype.toTree = function() {
...
-      include: ['ng2*/bundles/*.js'],
-      exclude: ['ng2*/bundles/*.min.js', 'ng2*/bundles/*.standalone.js'],
-    });
-
-    var thirdPartyJs = new Concat(thirdPartyJsTree, {
-      inputFiles: ['**/*.js'],
-      outputFile: '/thirdparty/libs.js',
-      allowNone: true
+    var typingsTree = new Funnel('typings', {
+      exclude: ['**/*.d.ts']
```
Comment (hansl) [inline lib/broccoli/angular2-app.js]: What does this tree _include_? `typings` should only contain `.d.ts` files.

Comment (jkuri) [inline lib/broccoli/angular2-app.js]: Its checking `typings` directory for changes but does not copy `.d.ts` files.

**Comment by hansl** [comment]
LGTM, please update and commit.

**Comment by filipesilva** [comment]
There might be some overlap between this PR and https://github.com/angular/angular-cli/pull/306

**Comment by jkuri** [comment]
Yes, closing this.

**Comment by jkuri** [comment]
Btw, I will include that in #299 so #306 wont be needed.

**Pull request closed without merging**